extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -505,6 +505,8 @@ export class AmpStoryPage extends AMP.BaseElement {
 
     this.mutateElement(() => {
       componentEls.forEach(el => {
+        userAssert(el.getAttribute('layout') === 'fixed', 'Embedded ' +
+          'components inside amp-story must use layout="fixed".');
```

Comment:
Is this going to be enforced at the validation level too?

Reply:
Not at validation, I tried doing it some time before but the current tooling doesn't allow it.

Reply:
I think we only allow `amp-audio` with a `layout=nodisplay` in stories, can we do something similar?
https://github.com/ampproject/amphtml/blob/master/extensions/amp-audio/validator-amp-audio.protoascii#L77-L94

We'd have to still allow any layout within an `amp-story-attachment` though.